### PR TITLE
Implement the Explore Page/issue62

### LINKED ISSUE #62: Implement the Explore Page
https://github.com/xkaper001/learn/blob/6e3e3a9d4dfae54b32d0e1dc1740402044a00538/lib/pages/explore.dart#L5

Implement the Explore page by adding all the avalable modules for the user here.

### PR DESCRIPTION
**Pull request by xkaper001**
Fixes #62

### CODE REVIEW

**Comment by xkaper001** [comment]
Hey @sapatevaibhav I have completed the #62 issue please review this.
Here's a demo of the implementation :)

https://github.com/VaibhavCodeClub/learn/assets/71162979/4f59dd09-a329-4ff6-a580-8405a20eb8c6

**Comment by sapatevaibhav** [comment]
Please resolve conflicts by fetching from latest

**Comment by K123Ritesh** [comment]
@xkaper001  Please Resolve The Conflics First

**Comment by sapatevaibhav** [comment]
Hello @xkaper001 as new merges are causing merge conflicts please fetch it as well add please quizzes page in explore page.

File: lib/pages/modules/colours.dart
Comment (K123Ritesh) [inline lib/pages/modules/colours.dart:1]: Minor Chage @xkaper001  change the color spelling

**Review by K123Ritesh — CHANGES_REQUESTED** [review]

**Comment by xkaper001** [comment]
> Hello @xkaper001 as new merges are causing merge conflicts please fetch it as well add please quizzes page in explore page. 

Okay @sapatevaibhav will do that

File: lib/pages/modules/colours.dart
Comment (xkaper001) [reply lib/pages/modules/colours.dart:1]: Hi @K123Ritesh , It was purposely kept that way to avoid misinterpretation with flutter `Colors`

Comment (K123Ritesh) [reply lib/pages/modules/colours.dart:1]: Ok

**Review by K123Ritesh — APPROVED** [review]

**Comment by xkaper001** [comment]
This PR already solves the newly created issue #97

**Pull request merged**